docs/how-to/data-plane-installation.md

```diff
@@ -0,0 +1,74 @@
+# Installing Kuadrant data-plane into an existing OCM Managed Cluster
+
+## Introduction
+This walkthrough will show you how to install and setup the Kuadrant Operator into an [OCM](https://open-cluster-management.io/) [Managed Cluster](https://open-cluster-management.io/concepts/managedcluster/).
```

Comment:
Is there a need to have two separate links for OCM and the Managed Cluster definition? Just the one pointing at https://open-cluster-management.io/concepts/managedcluster would work I think